### PR TITLE
Reaper cult sprite overhaul

### PR DESCRIPTION
**Pull request by SteelSlayer**
## What Does This PR Do
Changes nearly every reaper cult related sprite except for items like veil shifters, cult daggers etc. 
Sprites by @McRamon.

GIF

Complete list of sprite changed:

- The Reaper
- Boneshaper
- Draugr - now named "Golem"
- Wraith - now named "Envoy of Death"
- Psycopomp - now named "Necrophage"
- Shade - now named "Banshee"
- Tome
- Runed metal
- Archives
- Forge
- Altar
- Pylon
- Cult wall
- Cult floor
- Cult girder
- Cult airlocks
- The Reaper's spawn animation
- The Wraith's phase shift in, and phase shift out animation

Object path code changes:
I changed the path name for the archives and altar to be less confusing because it was really annoying me. 

I went through the code and replaced every instance I could find with their new path name
`/obj/structure/cult/tome` is now: `/obj/structure/cult/archives`
`/obj/structure/cult/functional/tome` is now: `/obj/structure/cult/functional/archives`

`/obj/structure/cult/talisman` is now: `/obj/structure/cult/altar`
`/obj/structure/cult/functional/talisman` is now: `/obj/structure/cult/functional/altar`

I feel like this makes it much easier to tell which structure you're working with.

## Why It's Good For The Game
The reaper sprites for cult felt outdated and we thought they could use an update. A lot of the sprites just re-used sprites from other things, such as the reaper floor being burnt carpet, the walls were just a giant mess of pixels, and so on.

## Images of changes

The new Reaper:
reaper

The Reaper's spawn animation:
reaperspawn

The Envoy of Death's phase out animation:
shadowstep_out

The Envoy of Death's phase in animation:
shadowstep_in

## Changelog
:cl: SteelSlayer and McRamon
add: Added a new subtype of shade used exclusively by cultists to prevent chaplains metagaming with their shade sprites
tweak: Renamed path names for the altar and archives to be less confusing, and replaced all instances of the path names to reflect the change
imageadd: Replaced nearly every sprite for reaper cult, including constructs, structures, airlocks and the reaper himself.
imagedel: Removed the old reaper cult construct sprites
/:cl:

### CODE REVIEW

**Comment by Couls** [comment]
REAPER SCYTHES FOR CULT DAGGERS

**Comment by Markolie** [comment]
Noooooice

**Comment by JKnutson101** [comment]
I've BEEN WAITING FOR THIS OVERHAUL! ❤️

**Comment by AzuleUtama** [comment]
Yes please.

**Comment by KasparoVy** [comment]
Looks absolutely fucking amazing.

File: code/game/gamemodes/cult/cult_datums.dm [hunk 1]
```diff
@@ -79,33 +106,60 @@
...
 
 	cult_wall_icon_state = "deathcult"
-	cult_floor_icon_state = "carpet-broken"
+	cult_floor_icon_state = "cultdeath"
+	cult_girder_icon_state = "reaper_cultgirder"
 
 	artificer_name = "Boneshaper"
+	artificer_icon_state = "boneshaper"
 
 	behemoth_name = "Draugr"
```
Comment (KasparoVy) [inline code/game/gamemodes/cult/cult_datums.dm]: Mob name wasn't changed, only the icon state.

**Comment by SteelSlayer** [comment]
@KasparoVy I was going to purely use `Initialize(mapload)`s for everything but I think I remember hearing if the parent does not use initialize and you use initialize on a child, some wonky bugs can happen. I'm may be wrong on this. But I basically looked at each object's parent, and if used `New()` I used new, if it used `Initialize()` I used initialize.

**Comment by KasparoVy** [comment]
Thanks for the response @SteelSlayer, I appreciate the explanation and marked my comment 'resolved'

**Comment by MrMagolor** [comment]
I'm glad that pylon code works as such that this won't work weird with the pylon changes I made

**Comment by Kyep** [comment]
Do wraiths, etc have identical mechanics no matter which cult they serve?
If so, it might be worth keeping "Wraith" in their name (and so on for the other mobs) so that people can more easily identify what they are.

**Comment by SteelSlayer** [comment]
@Kyep Yes, all constructs have identical behavior across all three cult themes, and the naming is more of a flavor thing. The current names for reaper cult and fire cult are already different from the original names. . I'd rather keep their names more thematic but I can change them to the standard names if people want.

**Comment by Fox-McCloud** [comment]
Maybe you *should* fear the reaper.

**Comment by variableundefined** [comment]
These are some really neat, high quality sprites

**Comment by MRgaty** [comment]
you don't get paid enough

**Pull request merged**